notebooks/initial_EDA.ipynb

```diff
@@ -8,10 +8,13 @@
     "\n",
```

Comment:
<p>Should we also add "../data/interim/correlation_sets.csv" to this PR?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
In general, I would like to keep data out of repositories and provide scripts for users to collect and process it themselves (see #10). But `correlation_sets.csv` is rather small and there is currently no instructions for how a user could generate it themselves, so I'll go ahead and add this output to the repo until that issue is addressed.